draft-tigress-requirements.md

```diff
@@ -165,8 +165,10 @@ sequenceDiagram
 
 # Assumptions
 
-- Depending on credential type, original credential information (with cryptographic key material) needs not to be sent. Instead, sender will be transferring its approval token for Receiver to acquire new credential information (e.g. receiver device may have just a subset of the sender's device access rights or receiver's device credential may be revoked independently of the sender's device credential).
--  Depending on credential type, Provisioning Entity will not allow for two users to use the same credential / cryptographic keys. Credentials on sender and receiver devices will be different and managed independently.
+- Depending on credential type, there are 3 possible scenarios for transferred credential cryptographic key material:
+  1) An existing cryptographic key is being copied and sent from Sender to receiver and provisioned to digital wallet as it is. In this case two credentails on both devices are indistinguashable. This scenario is currently used by a very limited number of entities.
+  2) Instead of the original cryptographic key, Sender device fetches a provisiong token from Provisiong entity approval and sends it to Receiver so that Receiver can acquire new credential information from Provisiong Entity presenting given provisiong token to it and receiving new cryptographic key material. In this case receiver device may have just a subset of the sender's device access rights or receiver's device credential may be revoked independently of the sender's device credential. As a result, Sender and Receiver devices have 2 different keys, often linked to the same logical "user account". Provisiong Entity will not allow the same cryptographic key be added to different physical devices.
```

Comment:
```suggestion
  2) Instead of the original cryptographic key, Sender device fetches a provisioning token from Provisioning entity approval and sends it to Receiver so that Receiver can acquire new credential information from Provisioning Entity presenting given provisioning token to it and receiving new cryptographic key material. In this case receiver device may have just a subset of the sender's device access rights or receiver's device credential may be revoked independently of the sender's device credential. As a result, Sender and Receiver devices have 2 different keys, often linked to the same logical "user account". Provisioning Entity will not allow the same cryptographic key be added to different physical devices.
```

Reply:
```suggestion
  2) Instead of the original cryptographic key, Sender device fetches a provisiong token from Provisiong entity approval and sends it to Receiver so that Receiver can acquire new credential information from Provisiong Entity presenting given provisiong token to it and receiving new cryptographic key material. In this case receiver device may have just a subset of the sender's device access rights or receiver's device credential may be revoked independently of the sender's device credential. As a result, Sender and Receiver devices have 2 different keys, often linked to the same logical "user account". Provisiong Entity will not allow the same cryptographic key to be added to different physical devices.
```

Reply:
fixed